frontend/ui/lib/services/in_memory_trip_service.dart

```diff
@@ -0,0 +1,23 @@
+import 'dart:async';
+import 'dart:core';
+import 'package:tripmeout/model/trip.dart';
+import 'package:tripmeout/services/trip_service.dart';
+
+class InMemoryTripService implements TripService {
+  final Map<String, Trip> tripsById = Map();
+
+  Future<List<Trip>> listTrips() async {
```

Comment:
do these functions not need override annotations?

Reply:
Good call.